oracle/Makefile

```diff
@@ -106,20 +106,21 @@ test:
 	bazel \
 		--bazelrc=controllers/inttest/.bazelrc \
 		test \
- 		--jobs=10 \
- 		--local_cpu_resources=10 \
- 		--test_tag_filters="integration" \
- 		... \
+		--jobs=10 \
+		--local_cpu_resources=10 \
+		--test_tag_filters="integration" \
+		... \
 		--test_output=errors \
 		--spawn_strategy=local \
 		--genrule_strategy=local \
 		--test_env=PROW_IMAGE_REPO=${PROW_IMAGE_REPO} \
 		--test_env=PROW_IMAGE_TAG=${PROW_IMAGE_TAG} \
 		--test_env=PROW_PROJECT=${PROW_PROJECT} \
 		--test_env=PROW_CLUSTER=${PROW_CLUSTER} \
+		--test_env=PROW_CANARY_JOB=${PROW_CANARY_JOB} \
 		--test_env=PROW_INT_TEST_SA=${PROW_INT_TEST_SA} \
-		--test_env=ARTIFACTS=${ARTIFACTS} \
-		--test_env=PROW_CLUSTER_ZONE=${PROW_CLUSTER_ZONE} || touch failed
+		--test_env=PROW_CLUSTER_ZONE=${PROW_CLUSTER_ZONE} \
```

Comment:
Was there any significance to reordering these?

Reply:
Just to keep all PROW_ envs together